modules/appeals_api/app/workers/appeals_api/higher_level_review_pdf_submit_job.rb

```diff
@@ -19,25 +19,24 @@ def generate_pdf(higher_level_review_id)
       pdf_constructor = AppealsApi::HigherLevelReviewPdfConstructor.new(higher_level_review_id)
       pdf_path = pdf_constructor.fill_pdf
       higher_level_review = HigherLevelReview.find higher_level_review_id
-      higher_level_review.processing!
+      higher_level_review.update!(status: 'processing')
       pdf_constructor.stamp_pdf(pdf_path, higher_level_review.consumer_name)
     end
 
     def upload_to_central_mail(higher_level_review_id, pdf_path)
       higher_level_review = AppealsApi::HigherLevelReview.find higher_level_review_id
       metadata = {
-        'veteranFirstName' => higher_level_review.auth_headers['first_name'],
-        'veteranLastName' => higher_level_review.auth_headers['last_name'],
-        'source' => higher_level_review.auth_headers['consumer_name'],
```

Comment:
i think this consumer line was a typo

Reply:
The line isn't a type as much as it's just referencing the wrong header here, the header might not be consumer_name or might be something different.
https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/appeals_api/app/controllers/appeals_api/v0/appeals_controller.rb#L46

```
'source' => higher_level_review.auth_headers['consumer_name'],
```